content/en/docs/ambient/usage/waypoint/index.md

```diff
@@ -10,25 +10,30 @@ test: no
 ---
 
 A **waypoint proxy** is an optional deployment of the Envoy-based proxy to add Layer 7 (L7) processing to a defined set of workloads.
-Waypoint proxies are installed, upgraded and scaled independently from applications; an application owner should be unaware of their existence. Compared to the sidecar data plane mode, which runs an instance of the Envoy proxy alongside each workload, the number of proxies required can be substantially reduced.
+
+Waypoint proxies are installed, upgraded and scaled independently from applications; an application owner should be unaware of their existence. Compared to the sidecar {{< gloss >}}data plane{{< /gloss >}} mode, which runs an instance of the Envoy proxy alongside each workload, the number of proxies required can be substantially reduced.
 
 A waypoint, or set of waypoints, can be shared between applications with a similar security boundary. This might be all the instances of a particular workload, or all the workloads in a namespace.
 
+As opposed to {{< gloss >}}sidecar{{< /gloss >}} mode, in ambient mode policies are enforced by the **destination** waypoint. In many ways, the waypoint acts as a gateway into the namespace. Istio enforces that all traffic coming into the namespace goes through the waypoint, which then enforces all policies for that namespace.
+
 ## Do you need a waypoint proxy?
 
-This layered approach of ambient allows users to adopt Istio in a more incremental fashion, smoothly transitioning from no mesh, to the secure overlay, to full L7 processing. If your applications require any of the following L7 mesh functions, you will need to use waypoint proxy for your applications:
+The layered approach of ambient allows users to adopt Istio in a more incremental fashion, smoothly transitioning from no mesh, to the secure L4 overlay, to full L7 processing.
 
-Most of the features of ambient mode are provided by the ztunnel node agent.  Ztunnel is scoped to only process traffic at Layer 4 (L4), so that it can safely operate as a shared component.
+Most of the features of ambient mode are provided by the ztunnel node agent. Ztunnel is scoped to only process traffic at Layer 4 (L4), so that it can safely operate as a shared component.
 
-When you add a waypoint proxy for a workload, traffic will be forwarded from the ztunnel to that waypoint. This enables the following features:
+When you add a waypoint proxy for a workload, traffic will be forwarded from the ztunnel to that waypoint. If your applications require any of the following L7 mesh functions, you will need to use a waypoint proxy:
```

Comment:
Taking the suggestion for the second half.  Was the first half ok?  Are you worried about the fact that it's not just about adding the waypoint, but the waypoint has to be the right traffic type?

Reply:
maybe it's just me getting too caught up in the details but waypoint-for will matter as well as how the traffic was addressed. I am working on a doc around this now though so maybe we can just move on and caveat/link to my stuff once it is ready

Reply:
well, adding to a workload would, technically speaking, require an appropriate waypoint for as well as the client to be addressing traffic to that workload instead of a service. maybe something that moves away from specifics is the call

```suggestion
When you configure redirection to a waypoint, traffic will be forwarded by ztunnel to that waypoint. If your applications require any of the following L7 mesh functions, you will need to use a waypoint proxy:
```